pysrc/main.py

```diff
@@ -0,0 +1,32 @@
+"""Main Entry point for app_hash sync"""
+import asyncio
+import os
+
+from dotenv import load_dotenv
+
+from pysrc.fetch.dune import DuneFetcher
+
+
+async def main(dune: DuneFetcher) -> None:
+    """App Hash Sync Logic"""
+    # TODO - use runtime config values here:
+    fname, column = "sync_block", "last_synced_block"
+    block_range = await dune.app_hash_block_range(fname, column)
+
+    hash_data = await dune.get_app_hashes(block_range)
```

Comment:
If this throws an error, do we still want to write the files in the next lines? Can you explain your solution?

Reply:
If this throws an error, the code will not reach the write steps.